.github/workflows/style.yml

```
@@ -60,6 +62,8 @@ jobs:
     #    fi
     - name: Shellcheck
       run: find . -type f -name '*.sh' -print0 | xargs -0 shellcheck
+    - name: Shell format (shfmt)
+      run: find . -type f -name '*.sh' -print0 | xargs -0 shfmt -d
```

```suggestion
      run: find . -type f -name '*.sh' -print0 | xargs -0 shfmt -d --shell-redirects
```
Could this be added here instead of adding a new file?

---

Technically yes but imo it is nicer to have the config in a file instead of having to worry about cli arguments while formatting.